tasks/e2e-kitchensink-eject.sh

```diff
@@ -116,8 +116,8 @@ REACT_APP_SHELL_ENV_MESSAGE=fromtheshell \
   npm run build
 
 # Check for expected output
-exists build/*.html
-exists build/static/js/main.*.js
+exists build/web/*.html
```

Comment:
Revert all changes in this file, and keep it consistent with CRA5,
Tao Fang is working on how to build and test CRA5